### ISSUE TITLE
Not following symbolic links

### ISSUE THREAD

**Issue by lepa22** [cboxdoerfer/fsearch#58]
I'm using fsearch in Ubuntu Mate 16.04 with Mate 1.16. I have a separate NTFS partition on my hard disk for my Documents, Music, Pictures and Videos folders and I create symbolic links of these folders in home. In fsearch I have the option for following symbolic links checked, but none of the aforementioned links are followed.

**Comment by cboxdoerfer** [comment]
Confirmed and I already know what's going wrong. Should be easy to fix.

**Comment by cboxdoerfer** [comment]
@lepa22, sorry I got bad news. I had to disable the follow symbolic links option for now. While fixing the bug you reported I ran into another issue where following symbolic links often resulted in endless loops and eventually the system running out of memory. Detecting such loops is possible and I've already tried one solution but that one was too slow.

So once I found a fast and reliable way to fix those issues I'll add the option again.

**Comment by lepa22** [comment]
Yeah, that's how I use symlinks since I started using fsearch. I was just always forgetting to report the bug.

Thanks for working on it!
